comms/dht/src/connectivity/mod.rs

```diff
@@ -67,6 +70,7 @@ pub struct DhtConnectivity {
     config: DhtConfig,
     peer_manager: Arc<PeerManager>,
     node_identity: Arc<NodeIdentity>,
+    ban_exclusions: Arc<RwLock<Vec<NodeId>>>,
```

Comment:
Can this not just be local? I think the other Arc's are shared amongst multiple different structs, whereas there should only be one DhtConnectivity

```suggestion
    ban_exclusions: Vec<NodeId>,
```